content/SCALE/SCALEUIReference/Credentials/DirectoryServices/ActiveDirectory.md

```diff
@@ -9,47 +9,75 @@ aliases:
-| **AD Timeout** | Number of seconds before timeout. To view the AD connection status, open the interface Task Manager. |
-| **DNS Timeout** | Number of seconds before a timeout. Increase this value if AD DNS queries time out. |
-| **Winbind NSS Info** | Choose the schema to use when querying AD for user/group info. *rfc2307* uses the Windows 2003 R2 schema support, *sfu* is for Service For Unix 3.0 or 3.5, and *sfu20* is for Service For Unix 2.0. |
-| **Netbios Name** | Netbios Name of this NAS. This name must differ from the Workgroup name and be no greater than 15 characters. |
-| **NetBIOS Alias** | Alternative names (no greater than 15 characters) that SMB clients can use when connecting to this NAS. Can be no greater than 15 characters. |
-| **Leave Domain** | Disconnects the TrueNAS system from the Active Directory. |
-
-{{< taglist tag="scaledirectoryservices" limit="10" title="Related Directory Services Articles" >}}
+| **Site Name** | Enter the Relative Distinguished Name (RDN) of the site object in the AD server. This is the first component of the **distingishedName** in AD. For more info read [Configuring Active Directory]({{< relref "ConfigADSCALE.md" >}}). |
+| **Kerberos Realm** | Select an existing realm from the dropdown list of options. Options are those configured in **Kerberos Realms**. After selecting **Enable (requires password or Kerberos principal)**, SCALE populates the **Kerberos Realm** and **Kerberos Principal** fields with what it discovered in AD. |
+| **Kerberos Principal** | Select the location of the principal in the keytab created in **Directory Services > Kerberos Keytabs**. After selecting **Enable (requires password or Kerberos principal)**, SCALE populates the **Kerberos Realm** and **Kerberos Principal** fields with what it discovered in AD. |
+|**Enable (requires password or Kerberos principle)** | Select to enable AD service. The first time you select this option you must enter the password for the domain admin account. After selecting **Enable (requires password or Kerberos principal)**, SCALE populates the **Kerberos Realm** and **Kerberos Principal** fields with what it discovered in AD. |
+| **Verbose Logging** | Select to log attempts to join the domain in <kbd>/var/log/messages</kbd>. |
+| **Allow Trusted Domains** | Selected if you do not want the username to include a domain name. Leave cleared to force the domain names to be prepended to usernames. One possible reason to not select this is to prevent username collisions when this is selected and there are identical usernames across multiple domains. |
```

Comment:
```suggestion
| **Allow Trusted Domains** | Select if you do not want the username to include a domain name. Leave cleared to force the domain names to be prepended to usernames. One possible reason to not select this is to prevent username collisions when this is selected and there are identical usernames across multiple domains. |
```